<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluriNotesClass</class>
 <widget class="QMainWindow" name="PluriNotesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PluriNotes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NoteViewer" name="noteViewer" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichiers</string>
    </property>
    <widget class="QMenu" name="menuNouveau">
     <property name="title">
      <string>Nouveau</string>
     </property>
     <addaction name="actionNote"/>
    </widget>
    <addaction name="menuNouveau"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="actionRelations"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNote">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
  <action name="actionDossier_de_stockage">
   <property name="text">
    <string>Dossier de stockage</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionRelations">
   <property name="text">
    <string>Relations..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NoteViewer</class>
   <extends>QWidget</extends>
   <header>NoteViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
